ambry-account/src/main/java/com/github/ambry/account/mysql/DatasetDao.java

```diff
@@ -149,10 +151,14 @@ public DatasetDao(MySqlDataAccessor dataAccessor, MySqlAccountServiceConfig mySq
             + "where (%s IS NULL or %s > now(3)) and %s = ? and %s = ? and %s = ? and %s = ? ORDER BY %s DESC LIMIT ?, 100",
         VERSION, DELETE_TS, DATASET_VERSION_TABLE, DELETE_TS, DELETE_TS, ACCOUNT_ID, CONTAINER_ID, DATASET_NAME,
         DATASET_VERSION_STATE, LAST_MODIFIED_TIME);
-    listValidDatasetVersionsSql = String.format("select %1$s from %2$s "
+    listValidDatasetVersionsByPageSql = String.format("select %1$s from %2$s "
             + "where (%3$s IS NULL or %3$s > now(3)) and %4$s = ? and %5$s = ? and %6$s = ? and %7$s = ? and %1$s >= ? "
             + "ORDER BY %1$s ASC LIMIT ?", VERSION, DATASET_VERSION_TABLE, DELETE_TS, ACCOUNT_ID, CONTAINER_ID, DATASET_NAME,
         DATASET_VERSION_STATE);
+    listValidDatasetVersionsByListSql = String.format("select %1$s, %8$s from %2$s "
```

Comment:
%8$s is same as %3$s. Can we remove one and use %3?

Reply:
Sure, updated.